Library/ViewModels/DiscoveryViewModelTests.swift

```diff
@@ -458,32 +458,32 @@ internal final class DiscoveryViewModelTests: TestCase {
     self.vm.inputs.sortPagerSelected(sort: .newest)
 
     XCTAssertEqual(
-      ["CTA Clicked", "CTA Clicked", "CTA Clicked"],
+      ["CTA Clicked", "CTA Clicked"],
```

Comment:
I keep seeing the number of events change from commit to commit. Could you explain what's happening? Should the older unit tests be emitting a different number of events with your changes?